src/Data.php

```diff
@@ -28,6 +37,22 @@ abstract class Data implements Arrayable, Responsable, Jsonable, EloquentCastabl
     use AppendableData;
     use ValidateableData;
 
+    public static function pipeline(): DataPipeline
+    {
+        // TODO: Should this be a factory?
```

Comment:
I'm unsure how I should interpret this comment. I'm assuming you mean to separate creational responsibilities from the data object?